specs/config-standard.md

```diff
@@ -0,0 +1,43 @@
+# Configuration Standard
+
+## Purpose
+
+Define how configuration should be implemented across all services within Astria services.
+
+## Background
+
+When running a service, it is generally [best practice to store configuration in environment variables](https://12factor.net/config), without defaults. This makes it difficult to commit environment specific config, and ensures separation of configuration and code. It makes it easier to integrate with KMS for secrets, and container orchestration systems are configured to enable this easily. Not having defaults ensures configuration is done with intention on a given environment.
```

Comment:
Very-very-nit: from my old latex days I prefer to use more linebreaks linebreaks so that adding a word or fixing a typo doesn't highlight the entire line.